saga-demo/saga-spring-demo/README.md

```diff
@@ -23,10 +23,14 @@ You can run the demo using either docker compose or executable files.
 ### via docker compose
 1. run the following command to create docker images in saga project root folder.
    ```
-   mvn clean package -DskipTests -Pdocker -Pdemo
+   mvn clean install -DskipTests -Pdocker -Pdemo
```

Comment:
package doesn't install the jars into maven directory.  We don't need to use install here.

Reply:
when first use,if you do not use mvn install in saga root project folder,it will got an error when use mvn help:evaluate in saga-demo.sh to get version of docker image.

[INFO] Scanning for projects...
[INFO] mixin-maven-plugin: Merging Mixins
[ERROR] Cannot resolve mixin artifact org.apache.servicecomb.saga:docker-build-config:pom:0.3.0-SNAPSHOT: Error resolving artifact org.apache.servicecomb.saga:docker-build-config:pom:0.3.0-SNAPSHOT -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException

Reply:
OK，we can keep the change here.